code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -126,10 +126,11 @@
 		return src.close()
 	return src.open()
 
-// this should probably use dump_contents()
 /obj/structure/closet/ex_act(severity, target)
 	contents_explosion(severity, target)

```

Comment:
![2015-02-08_19-50-22](https://cloud.githubusercontent.com/assets/3855300/6097836/e6fd81a6-afcb-11e4-927a-f98c5b0c7307.gif)

it still works